traitsui/editors/tuple_editor.py

```diff
@@ -73,9 +58,10 @@ class ToolkitEditorFactory(EditorFactory):
     # 'enter_set' metadata or an editor defined.
     enter_set = Bool(False)
 
-#-------------------------------------------------------------------------
-#  'SimpleEditor' class:
-#-------------------------------------------------------------------------
+    # The validation function to use for the Tuple. If the edited trait offers
+    # already a validation function then the value of this trait will be
```

Comment:
As you say it is not analogous. Since there is no restriction on the label. However if the Editor validation overrides the trait validation it can potentially result in an invalid value been forced into a trait (i.e. a ValidatedTuple trait) which will throw an exception.